blueprint/core/blueprint_ReactApplicationRoot.h

```diff
@@ -34,238 +34,25 @@ namespace blueprint
     class ReactApplicationRoot : public View
     {
     public:
-        //==============================================================================
-        // We allow registering arbitrary view types with the React context by way of
-        // a "ViewFactory" here which is a user-defined function that produces a View
-        // and a corresponding ShadowView.
-        typedef std::pair<std::unique_ptr<View>, std::unique_ptr<ShadowView>> ViewPair;
-        typedef std::function<ViewPair()> ViewFactory;
-
         //==============================================================================
         ReactApplicationRoot()
         {
             JUCE_ASSERT_MESSAGE_THREAD
 
+            //TODO: We used to construct the root ShadowView after calling registerNativeRenderingHooks.
+            //      Is there any issue moving the construction of the root ShadowView here?
```

Comment:
This appears to be all good from my testing but worth you sanity checking @nick-thompson